ukvm/ukvm_hv_kvm_x86_64.c

```diff
@@ -182,8 +182,8 @@ void ukvm_hv_vcpu_loop(struct ukvm_hv *hv)
 
         switch (run->exit_reason) {
         case KVM_EXIT_HLT:
```

Comment:
Please remove this entire `case` block, so that execution falls through to `default:` below.

Reply:
changed